FAQ.md

```diff
@@ -0,0 +1,113 @@
+Here's the translated FAQ document in English, formatted in markdown for better readability:
```

Comment:
Is this a mistake? I guess it's response from a LLM, and it should be deleted

Reply:
Yes, fixed. @Yeuoly